addons/point_of_sale/static/src/js/Popups/CashOpeningPopup.js

```diff
@@ -1,63 +1,44 @@
 odoo.define('point_of_sale.CashOpeningPopup', function(require) {
     'use strict';
 
-    const { useState, useRef} = owl.hooks;
-    const { useListener } = require('web.custom_hooks');
-    const NumberBuffer = require('point_of_sale.NumberBuffer');
-    const PosComponent = require('point_of_sale.PosComponent');
+    const { useState, useRef } = owl.hooks;
     const AbstractAwaitablePopup = require('point_of_sale.AbstractAwaitablePopup');
     const Registries = require('point_of_sale.Registries');
+    const { round_precision: round_pr } = require('web.utils');
 
 
     class CashOpeningPopup extends AbstractAwaitablePopup {
         constructor() {
             super(...arguments);
-            this.cashBoxValue = this.env.pos.bank_statement.balance_start || 0;;
             this.currency = this.env.pos.currency;
```

Comment:
This doesn't seem to be used in the component. It can be removed.

Reply:
Yeah indeed, it was before the big refactor